banzai/settings.py

```diff
@@ -1,79 +1,102 @@
-import sys
-import logging
 import operator
-
-from lcogt_logging import LCOGTFormatter
+from abc import ABC
 
 from banzai.context import TelescopeCriterion
 from banzai import qc, bias, crosstalk, gain, mosaic, bpm, trim, dark, flats, photometry, astrometry, images
 
 
-# Logger set up
-logging.captureWarnings(True)
+class Settings(ABC):
+
+    @property
```

Comment:
I think you want to use the abstractmethod here instead of a not implemented error.

Reply:
Fixed